tests/test_resolutions.py

```diff
@@ -166,15 +165,14 @@ def test_the_test(self) -> None:
 
     def test_description(self) -> None:
         test_entry = {"spam": "eggs"}
-        test_entries = {"tree": "larch", "spam": "eggs"}
+        test_entries = {"number": 1234, "spam": "eggs"}
 
         cte_single = ContainsTheEntry(**test_entry)
         cte_multiple = ContainsTheEntry(**test_entries)
 
-        expected_description_single = "A mapping with the entry spam->eggs."
+        expected_description_single = "A mapping with the entry 'spam'->'eggs'."
         expected_description_multiple = (
-            "A mapping with the entries"
-            f" {', '.join(f'{k}->{v}' for k, v in test_entries.items())}."
+            "A mapping with the entries 'number'-><1234>, 'spam'->'eggs'."
```

Comment:
Thank you, this is much more readable than what i had!

Reply:
Do you have concerns about that arrow pointing to `<1234>`? It looks a little weird to me.
```
'number'-><1234>
```

Reply:
That's how ints are represented in hamcrest.  At first I thought it was odd too.   But it does eliminate _any_ mistaking that value for a string.  /shrug